src/karmabot/settings.py

```diff
@@ -6,11 +6,14 @@
 from dotenv import load_dotenv
 
 
-def _get_env_var(env_var: str):
+def get_env_var(env_var: str, default: str = None) -> str:
```

Comment:
Had to make this public as I imported it into welcome.py

Reply:
Btw why do we have this function? I would expect python-dotenv, python-decouple can do all this?

Reply:
#43

Reply:
yeah ok :)